README.md

```diff
@@ -1 +1,140 @@
-# drupal-11-template
+# Drupal 11 TEMPLATE
+
+This is a Drupal project template for starting new projects. The template
+includes default setup and a starting point for:
+
+- PHP Code sniffer
+- PHP Code styles
+- Github actions
+- Tasks
+- Local settings
+- Docker and overrides
+- Readme
+- Changelog
+
+## Usage
+
+### Build new project from template
+
+Build a new project using this template
+
+``` shell
+gh repo create itk-dev/{PROJECT-NAME} --template itk-dev/drupal-11-template --public --clone 
+```
+
+### Prune the project
+
+Prune the project to remove/override template specific files and setup the
+project, with docker config and settings.
+
+``` shell
+task template-prune PROJECTNAME={PROJECTNAME}
+```
+
+This will:
+
+- Remove TEMPLATE_THEME and cause coding-standards-check to fail until a new
+theme is added.
+- Add/override .env and .task.env files with files that uses {PROJECTNAME}
+- Add/override a settings.local.php to web/sites/default folder
+- Add/override a services.local.yml to web/sites/default folder
+- Override CHANGELOG.md
+- Override Taskfile.yml
+
+### Finish
+
+- Replace {PROJECT-NAME} placeholder in this README.md and CHANGELOG.md
+
+<!-- markdownlint-disable MD025 -->
+
+### ---- After building a project delete this line and all above ----
```

Comment:
The `template-prune` task could to this.

Reply:
Just a matter of time...